install-deps.sh

```diff
@@ -65,7 +65,7 @@ function munge_debian_control {
 	sed -i -e 's/^# Crimson[[:space:]]//g' $control
     fi
     if $with_jaeger; then
-	sed -i -e 's/^# Jaeger[[:space:]]//g' $control
+	sed -i -e 's/^# Jaeger[[:space:]]\+//g' $control
```

Comment:
@clwluvw could you move this commit into #38743 ?

Reply:
@tchaikov Sure! Done.